example/gan/CGAN_mnist_R/CGAN_train.R

```diff
@@ -167,6 +173,7 @@ for (iteration in 1:2400) {
   }
 }
 
+dir.create(file.path(".", "models"))
```

Comment:
Call `dir.exists` first.

Reply:
if the folder already exists, it will just throw a warning message.

Reply:
Yes, but it's still better to have no warning in a ready-to-run example.